charts/access/discord/README.md

```diff
@@ -0,0 +1,101 @@
+# Teleport Access Request Discord Plugin
+
+This chart sets up and configures a Deployment for the Access Request Discord plugin.
+
+## Installation
+
+See the [Access Requests with Discord guide](https://goteleport.com/docs/access-controls/access-request-plugins/ssh-approval-discord/)
```

Comment:
This link returns 404
I assume we'll a PR with proper instructions later on

Reply:
👍 https://github.com/gravitational/teleport/pull/17593